check_buyable_vs_available.py

```
@@ -0,0 +1,97 @@
+"""
+매수가능금액 vs 인출가능금액 비교
+"""
+import requests
+from token_manager import TokenManager
+from config import *
```

Ruff (0.14.1)

`from config import *` used; unable to detect undefined names

(F403)